utils/fish/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2017 Shane Peelar
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+BASE_VERSION:=2.5.0
+
+PKG_NAME:=fish
+PKG_VERSION:=$(BASE_VERSION)
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(BASE_VERSION).tar.gz
+PKG_SOURCE_URL:=https://fishshell.com/files/$(BASE_VERSION)/
+PKG_SHA1SUM:=c5981245b45ea61b765008299b5ac477657089c3
```

Comment:
PKG_SHA1SUM is not supported please use PKG_HASH:=<sha256sum>

Reply:
Fixed